src/atopile/front_end.py

```diff
@@ -768,6 +558,18 @@ def _import_item(
 
         from_path = self._sanitise_path(candidate_from_path)
 
+        # TODO: @v0.4: remove this shimming
+        import_addr = address.AddrStr.from_parts(from_path, ".".join(item.ref))
+        for shim_addr, (shim_cls, preferred) in shim_map.items():
+            if import_addr.endswith(shim_addr):
+                with downgrade(DeprecationError):
+                    raise DeprecationError.from_ctx(
+                        item.original_ctx,
+                        f"Deprecated: {import_addr} is deprecated and a likeness"
+                        f" is being shimmed in place for you. Use {preferred} instead.",
```

Comment:
Lines too long then, isn't it? I'd prefer to keep these within spec for the minute to avoid needing to manually fix strings later - since the ruff checks don't do it I believe